README.md

```diff
@@ -89,6 +89,13 @@ Display value by key.
  * `KEY` - The key
  * `FORMAT` - Data Display Format (Optional)
 
+### export
+> export `KEY` `FILENAME`
+
+Write value by key to .jpg file with specified filename.
```

Comment:
Why only the extension ".jpg"?
user must be able to specify any file name or path.
Please change to:
```
Write value by key to file with specified filename or path.
```

Reply:
done